File: RUST/Cap03/c03ex02.rs
// c03ex02.rs

use std::f64::consts::PI;
use std::io::{self, Write};

fn main() {
  let r: f64;
  let a: f64;

  print!("Entre valor do raio: ");
  io::stdout().flush().expect("Falha ao limpar o buffer");
  let mut entrada = String::new();
  io::stdin().read_line(&mut entrada).expect("Falha ao ler a entrada");
  r = entrada.trim().parse().expect("Entrada inválida");

  a = PI * r.powf(2.0);

  println!("Resultado = {:.2}", a);

  println!();
  print!("Tecle <Enter> para encerrar... ");
  io::stdout().flush().expect("Falha ao limpar o buffer");
  let mut _entrada = String::new();
  io::stdin().read_line(&mut _entrada).expect("Falha ao ler a entrada");
}

File: VB_NET/Cap03/c03ex02.vb
' c03ex02.vb

Imports System
Imports System.Console

Module Program
    Sub Main(args As String())
        Dim R As Double
        Dim A As Double

        Write("Entre valor do raio: ")
        R = Double.Parse(ReadLine())

        A = Math.PI * Math.Pow(R, 2)

        WriteLine("Resultado = {0:F2}", A)

        Write(vbCrLf + "Aperte <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
